data/transactions/logic/langspec_v1.json

```diff
@@ -1,6 +1,6 @@
 {
   "Version": 1,
-  "LogicSigVersion": 9,
+  "LogicSigVersion": 10,
```

Comment:
Looking at this now, I suppose the fix is to remove `LogicSigVersion` entirely. I don't see the value with `Version` right there doing the same thing.  But I think we said we'd do that in master.

Reply:
I think we landed on doing it in master, agreed.